### PR TITLE
fix: remove deprecated swagger-cli

### PR DESCRIPTION
**Pull request by abelanger5**
# Description

Removes the `swagger-cli` dependency that was causing issues during builds. 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: api/v1/server/oas/transformers/workflow.go [hunk 1]
```diff
@@ -148,7 +148,7 @@ func ToWorkflowVersionConcurrency(concurrency *WorkflowConcurrency) *gen.Workflo
 
 	res := &gen.WorkflowConcurrency{
 		MaxRuns:             concurrency.MaxRuns.Int32,
-		LimitStrategy:       gen.WorkflowConcurrencyLimitStrategy(concurrency.LimitStrategy.ConcurrencyLimitStrategy),
+		LimitStrategy:       gen.ConcurrencyLimitStrategy(concurrency.LimitStrategy.ConcurrencyLimitStrategy),
```
Comment (hatchet-temporary) [inline api/v1/server/oas/transformers/workflow.go:151]: is this change related?

**Review by hatchet-temporary — APPROVED** [review]
LGTM 👍

File: api/v1/server/oas/transformers/workflow.go (hunk 1 above)
Comment (abelanger5) [reply api/v1/server/oas/transformers/workflow.go:151]: Yes, somehow changing the package for bundling changed our generated API as well. Seems to not be an issue but will double check on staging.
